src/Raven.Studio/wwwroot/App/views/shell/about.html

```diff
@@ -55,6 +55,22 @@
                 </div>
             </div>
         </div>
+        <div data-bind="visible: passiveNode">
+            <div class="bg-info text-info flex-horizontal padding">
+                <div class="flex-start"><i class="icon-info"></i></div>
+                <div>The running server is in a <strong>Passive State</strong>, it is not part of a cluster yet.<br>
+                     In order to work, it is required that the server is part of a cluster.<br><br>
+                    Either one of the following can be done to <strong>Bootstrap a Cluster</strong>:<br>
+                     <ul>
+                         <li>Create a new database</li>
+                         <li>Register a license (if not registered yet)</li>
```

Comment:
do we know if license is registered or no?

Reply:
"In order to work" - what's not working?

Reply:
> do we know if license is registered or no?

As far as I see, at this point, we don't have this info
The '`registered`' observable https://github.com/ravendb/ravendb/blob/efba92603d4ba571907188e038c03b0b91250e13/src/Raven.Studio/typescript/viewmodels/shell/about.ts#L189
returns relevant info **only** when we have a cluster